examples/operator/operator.yaml

```diff
@@ -2,8 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.3
-  creationTimestamp: null
+    controller-gen.kubebuilder.io/version: v0.16.1
```

Comment:
@frouioui Shall only add the HPA related changes in this PR to make the diff smaller?

Reply:
I think it's better to copy everything. If there is a larger diff it's because we lost sync at some point, but we should always be in sync :)